RA-Policy.md

```diff
@@ -20,4 +20,14 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md). 
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+All GSA teams, being part of a federal agency, follow the risk assessment and management process outlined in [NIST Special Publication (SP) 800-37](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-37r1.pdf), _Guide for Applying the Risk Management Framework to Federal Information Systems_.  
+
+Initial security categorization is a collaborative and inter-disciplinary activity, with all final decisions made by the Authorizing Official. Risk assessment is similarly cross-functional, with the Cloud Operations team being primarily responsible, as directed by the Program Manager. GSA Information Security teams are consulted throughout, and the Authorizing Official seeks concurrence from the GSA Chief Information Security Officer whenever possible. The Authorizing Official seeks to minimize any need to issue waivers or risk acceptances from GSA IT Security policy (see Chapter 1, Section 5) that do not have concurrence from the GSA Chief Information Security Officer.
+
+See RA-2, RA-3.
+
+Cloud Operations and GSA Information Security work together to scan all of relevant portions of the cloud.gov stack. This includes any customer controls inside of AWS GovCloud, the operating system baseline of AWS EC2 instances, the baseline of containers, the code of Cloud Foundry, and any 18F built or implemented cloud.gov modules. Access to scanning tools, scan results, and logs is broadly shared amongst the cloud.gov team to ensure a rapid response to any findings. Similarly, on-demand access is granted to the Authorizing Official to aide in any systemic understanding of the system's risk posture.
```

Comment:
I wonder if "Access to scanning tools, scan results, and logs is broadly shared amongst the cloud.gov team" should be refined to reflect that not everyone on the cloud.gov team gets access to run these tools or look at raw results/logs. We could say that access to these are broadly shared among GSA Information Security and Cloud Operations, and they can discuss the tools/results/logs with the rest of the cloud.gov team?